src/test/resources/features/Registration.feature

```diff
@@ -0,0 +1,14 @@
+@all @reg @accessibility @zap
+Feature: Register for ECL service
+
+  Scenario Outline: User able to select their UK revenue category for ECL service registration
+    Given I am on the registration start page
+    When I click on Start now button
+    And I enter valid login credentials
```

Comment:
As discussed - these test packs should initially be automated against service manager / local, not QA

we can consider some automated QA smoke tests in the future once it is in a bit more of a stable/known state - but this relies on real data.

Reply:
changed the code to run against local environment